doc/classes/LinkButton.xml

```diff
@@ -12,8 +12,13 @@
 	</methods>
 	<members>
 		<member name="text" type="String" setter="set_text" getter="get_text" default="&quot;&quot;">
+			Sets the label of the [LinkButton] to the specified text.
 		</member>
 		<member name="underline" type="int" setter="set_underline_mode" getter="get_underline_mode" enum="LinkButton.UnderlineMode" default="0">
+			Sets the [code]underline[/code] mode of the [linkButton]. See [LinkButton.UnderlineMode] for the available modes.
```

Comment:
Should be `[LinkButton]`, not camelCase.

No need to make `underline` code-formatted, "Sets the underline mode" is clear enough under a code-formatted `underline` property. Same comments for `link` property.

Reply:
There's also no need to refer to `[LinkButton]` in all descriptions. It adds many unnecessary hyperlinks as we are already in LinkButton's own documentation, so it's clear from the content that `text` sets the label of the current node.